package.json

```diff
@@ -30,23 +28,24 @@
     "babel-plugin-transform-object-rest-spread": "^6.26.0",
     "babel-preset-env": "^1.7.0",
     "babel-register": "^6.26.0",
-    "eslint": "^4.14.0",
+    "eslint": "^4.19.1",
```

Comment:
This should be updated

Reply:
and I would 100% agree with you. Let me dot it